pbf/models.py

```diff
@@ -360,6 +360,12 @@ def elements(self):
         counter[Elements.Earth] += self.temporary_earth + self.permanent_earth
         counter[Elements.Plant] += self.temporary_plant + self.permanent_plant
         counter[Elements.Animal] += self.temporary_animal + self.permanent_animal
+
+        if self.aspect == 'DarkFire':
+            counter[Elements.Moon] += 1
+        elif self.aspect == 'Intensify':
```

Comment:
I do understand this can be done with something like `self.aspect in ('DarkFire', 'Intensify')`, but I chose to keep each aspect on one line for a clear diff and for obvious extension to future aspects with elements